InterfaceParser/model/struct.py

```diff
@@ -0,0 +1,38 @@
+"""Structure.
+
+"""
+
+from collections import OrderedDict
+
+from model.interface_item_base import InterfaceItemBase
+
+
+class Struct(InterfaceItemBase):
+    """Structure.
+
+    :param name: item name
+    :param description: list of string description elements
+    :param design_description: list of string design description elements
+    :param issues: list of issues
+    :param todos: list of string todo elements
+    :param platform: optional platform (string or None)
+    :param scope: optional scope: internal, partner or none (none by defaul, means public)
```

Comment:
default* 

Check for all instances of that typo, seems to bea  copy paste error

Reply:
fixed typo for all instances